Utils/Dataflow/pyDKB/storages/client/Client.py

```diff
@@ -0,0 +1,34 @@
+"""
+pyDKB.storages.client.Client
+"""
+
+from pyDKB.common import LoggableObject
+
+
+class Client(LoggableObject):
+    """ Interface class for external and internal DKB storage clients. """
+
+    def __init__(self):
+        """ Initialize Storage object. """
+        raise NotImplementedError
+
+    def configure(self, cfg):
+        """ Apply storage configuration (initialize client).
+
+        :param cfg: configuration parameters
+        :type cfg: dict
+        """
+        raise NotImplementedError
+
+    def get(self, oid, **kwargs):
+        """ Get object / record from storage by ID.
+
+        Raise ``NotFound`` exception if object / record not found.
```

Comment:
"... record WAS not found."?

Reply:
Docstrings are more about providing necessary information than to be a literary writing; so "terse language" looks appropriate for me, even though it is not a error message or newspaper headline (check related discussion [here](https://english.stackexchange.com/a/73369)). It may be a matter of habit (and my bad taste, possibly ;) ), but I don\`t think it should be changed.

However if you think that we should set some common rule for such cases, I am open to the further discussion :) (But better not in this PR, as it is not about this one only, but more about common style.)